.github/workflows/test-workflow.yml

```diff
@@ -16,6 +16,9 @@ jobs:
       - name: 현재 날짜 출력
         run: echo "현재 시간은 $(date) 입니다."
 
+      - name: 브랜치 정보 출력
+        run: echo "현재 실행 중인 브랜치 - ${{ github.ref }}"
+
       - name: 간단한 스크립트 실행
         run: |
           echo "여러 줄 명령어 실행 예시:"
```

Comment:
The addition of branch information output is a good practice for debugging. Consider also outputting `github.head_ref` if the workflow is triggered by a pull request, as `github.ref` will refer to the merge ref in that case.